docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+	- [PDN Configuration](#PDNConfiguration)
+	- [Tapcell Configuration](#TapcellConfiguration)
+	- [setRC Configuration](#setRCConfiguration)
+	- [KLayout](#KLayout)
+		- [KLayout properties file](#KLayoutpropertiesfile)
+		- [KLayout tech file](#KLayouttechfile)
+- [Validating the New Platform](#ValidatingPlatform)
+- [Authors/Contributors](#AuthorsContributors)
+
+
+# Overview <a name="Overview"></a>
+
+This document is a guide for foundry and third party IP providers to easily integrate and test a new technology into the OpenROAD RTL to GDS flow. OpenROAD allows you to integrate any public PDK (Process Design Kit) for any feature size and implement a fully open-sourced RTL-GDSII flow (synthesizable Verilog to merged GDSII). The OpenROAD flow has been validated for feature sizes up to 7nm and used to design and tapeout over 100 ASIC and SoCs to date.
+
+
+# Prerequisites <a name="Prerequisites"></a>
+
+To build and add a new platform for OpenROAD, key technology and library components must be provided based on the technology node. These are generally available as part of the standard design kit provided by a foundry or a third-party IP provider.  
+
+They include :
+
+* A standard cell library
+    * GDS files of all standard cells in the library (or a way to generate them from the layout files, eg. Magic VLSI layout tool)
+* A technology LEF file of the PDK being used that includes all relevant information regarding metal layers, vias, and spacing requirements. 
+    * Can be generated through the commercial tool Cadence Abstract.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lef/NangateOpenCellLibrary.tech.lef` as an example tech LEF file.
+* A macro LEF file of the standard cell kit that includes MACRO definitions of every cell, pin designations (input/output/inout), and size data of metal layers with port connections.
+    * Can be generated through the commercial tool Cadence Abstract.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lef/NangateOpenCellLibrary.macro.lef `as an example macro LEF file.
+* A Liberty file of the standard cell library with PVT characterization, input and output characteristics, timing and power definitions for each cell.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lib/NangateOpenCellLibrary_typical.lib `as an example liberty file.
+
+Adding a new platform additionally requires the following: 
+
+* A validated installation of the OpenROAD flow scripts is available. See instructions [here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html#building-and-installing-the-software).:
+* A general knowledge of VLSI design and RTL to GDS flows.  \
+OpenROAD implements a fully-automated RTL-GDSII but it requires familiarity with the OpenROAD flow scripts flow and the scripts provided to enable debugging in case of problems.Add a flow section briefly describing the OR or ORFS flow steps to add the platform a platform here
```

Comment:
"Add a flow section briefly describing the OR or ORFS flow steps to add the platform a platform here"

looks like a todo.  Is this ready for merging?